spec/unit/typesafe/config/conf_parser_spec.rb

```diff
@@ -781,20 +781,26 @@ def assert_comments_at_path_index(comments, conf, path, index)
   # Skipping 'includeURLHeuristically' because we don't support URLs
   # Skipping 'includeURLBasenameHeuristically' because we don't support URLs
 
-  it "acceptBOMStartingFile" do
-    skip("BOM not parsing properly yet; not fixing this now because it most likely only affects windows") do
-      # BOM at start of file should be ignored
-      conf = Hocon::ConfigFactory.parse_file(TestUtils.resource_file("bom.conf"))
-      expect(conf.get_string("foo")).to eq("bar")
-    end
+  it "doesnotacceptUTF8FileNames" do
+    expect { Hocon::ConfigFactory.parse_file(TestUtils.resource_file("ᚠᛇᚻ.conf")) }.to raise_error
   end
```

Comment:
The nice thing about actually running / asserting a behavior, is that when the library changes, the test automatically breaks and you're forced to repair it.

With `skip`, it can be easy to miss that.  However, if you're looking to grep for `pending`, then that's another story.  It's your lib, so I'm fine with changing it to your preference.